ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/panel/PieceCrafterDetailPanel.java

```diff
@@ -468,7 +468,7 @@ private void initGUIComponents(PieceTypeBuilder builder) {
         constraints.gridwidth = 1;
         constraints.gridx = 0;
         constraints.gridy = 7;
-        add(cancelButton, constraints);
+        //add(cancelButton, constraints);
```

Comment:
Not sure if it is needed or not, do we need people to be able to cancel their changes to a piece using a specific button?